test_scripts/AppServices/RPCPassThrough/001_SendLocation_Handled_Request.lua

```diff
@@ -63,24 +36,51 @@ local successResponse = {
   info = "Request was handled by app services"
 }
 
-local sendLocationRequest = { 
-  name = "SendLocation",
-  hmi_name = "Navigation.SendLocation",
+local rpcRequest = { 
   params = {
     longitudeDegrees = 50,
     latitudeDegrees = 50,
     locationName = "TestLocation" 
   },
-  hmi_params = params
+  hmi_params = {
+    longitudeDegrees = 50,
+    latitudeDegrees = 50,
+    locationName = "TestLocation" 
+  }
```

Comment:
Eh, if `params` and `hmi_params` aren't different, the `params` field should just be used in both cases. I'd only split them up if they are actually different.

Reply:
Since I'm planning to add the rpc_name in the struct as well it could be used for a different RPC. In which case the hmi_params could be different for that rpc

Reply:
> In which case the hmi_params could be different for that rpc
correct, but it isn't in this case.